Doc/library/enum.rst

```diff
@@ -425,17 +426,21 @@ Data Types
    in most of the same places that a string can be used.  The result of any string
    operation performed on or with a *StrEnum* member is not part of the enumeration.
 
-   .. note:: There are places in the stdlib that check for an exact :class:`str`
-             instead of a :class:`str` subclass (i.e. ``type(unknown) == str``
-             instead of ``isinstance(unknown, str)``), and in those locations you
-             will need to use ``str(StrEnum.member)``.
+   .. note::
+
+      There are places in the stdlib that check for an exact :class:`str`
+      instead of a :class:`str` subclass (i.e. ``type(unknown) == str``
+      instead of ``isinstance(unknown, str)``), and in those locations you
+      will need to use ``str(StrEnum.member)``.
 
    .. note::
 
       Using :class:`auto` with :class:`StrEnum` results in the lower-cased member
       name as the value.
 
-   .. note:: :meth:`__str__` is :func:`str.__str__` to better support the
+   .. note::
+
+      :meth:`__str__` is :func:`str.__str__` to better support the
```

Comment:
```suggestion
      :meth:`~object.__str__` is :meth:`!str.__str__` to better support the
```

* Fix broken link to abstract `__str__`
* Don't attempt to resolve `str.__str__` to avoid a failed resolution and a warning
* Use appropriate role on `str.__str__`